CHANGES.md

```diff
@@ -1,3 +1,35 @@
+## edge-19.8.3
+
+This edge release introduces a new `linkerd stat trafficsplits` subcommand, to
+show traffic split metrics. It also introduces a "Kubernetes cluster monitoring"
+Grafana dashboard.
+
+* CLI
+  * Added traffic split metrics via `linkerd stat trafficsplits` subcommand
+  * Fixed `linkerd uninject` not removing `linkerd.io/inject: enabled`
+    annotations
+  * Fixed `linkerd stat -h` example commands (thanks @ethan-daocloud!)
+* Controller
+  * Removed Tap from the Public API, now only available via authenticated TAP
```

Comment:
How about: Removed old non-authenticated endpoints for `linkerd tap`, which is now only available via the authenticated API?

Reply:
Actually both that wording and your suggestion are not quite true. They will be in stable-2.5. For now changing this to:
`Removed unauthenticated Tap from the Public API`

Reply:
Just... make the capitalization make me happy